GW2EIParser/MainForm.cs

```diff
@@ -877,4 +892,26 @@ private void EnableSettingsWatcher(object sender, EventArgs e)
         AddTraceMessage("Settings: Closing settings");
         BtnSettings.Enabled = true;
     }
+
+    private void BtnCheckUpdates_Click(object sender, EventArgs e)
+    {
+        AddTraceMessage("Updater: Checking for updates");
+
+        Task.Factory.StartNew(async () =>
+        {
+            Updater.UpdateInfo info = await Updater.CheckForUpdate("GW2EI.zip");
+            if (info.UpdateAvailable)
```

Comment:
The if and else should update UpdateAvailable/UpdateLastChecked settings and refresh the label text